generator.rb

```diff
@@ -0,0 +1,29 @@
+def get_the_vowel(word)
+  word.length > 1 && word =~ /[aeiou]/
+end 
+
+puts "Please enter your first word"
+first_word = gets.chomp.downcase
+ until get_the_vowel(first_word)
+  puts "Please enter a word with 2 or more characters and a vowel"
+  first_word = gets.chomp.downcase
+  end
+ 
+words = []
+a = first_word.rindex(/[aeiou].*/)
```

Comment:
again, would love a comment on this part. Most programmers don't choose to solve problems with regular expressions because they are hard to read on their own even if they are very efficient.